controllers/sr_controller/sr/robot/ruggeduino_devices.py

```diff
@@ -46,3 +46,17 @@ def __init__(self, webot, sensor_name):
 
     def read_value(self):
         return self.webot_sensor.getValue() > 0
+
+
+class Led(object):
+
+    def __init__(self, webot, device_name):
+        self.webot = webot
+        self.device_name = device_name
+        self.webot_sensor = self.webot.getLED(self.device_name)
+
+    def read_value(self):
+        return self.webot_sensor.get() > 0
```

Comment:
What does it mean to read a value from an LED?

Reply:
Arduino allows you to read back the output pin, for example quick pin state toggling:
> digitalWrite(pin, !digitalRead(pin))

I'm not familiar with Ruggeduinos firmware, but it's a cool trick.